content/docs/command-reference/remote/modify.md

```diff
@@ -464,18 +464,22 @@ a full guide on using Google Drive as DVC remote storage.
   using a custom Google Client project. Also requires using `gdrive_client_id`.
 
   ```dvc
-  $ dvc remote modify myremote \
+  $ dvc remote modify --local myremote \
```

Comment:
here?

Reply:
Yes? (Not sure what the Q was, sorry.) This is for `gdrive_client_secret`.

Reply:
`gdrive_client_secret` is also fine to include